seaborn/_statistics.py

```diff
@@ -345,3 +345,39 @@ def __call__(self, x1, x2=None, weights=None):
             return self._eval_univariate(x1, weights)
         else:
             return self._eval_bivariate(x1, x2, weights)
+
+
+class ECDF:
+
+    def __init__(self, stat="proportion"):
+
+        _check_argument("stat", ["count", "proportion"], stat)
+        self.stat = stat
+
+    # Do we need bivariate ECDF?
```

Comment:
I think the bivariate visualization will be difficult to explain, so I think a univariate is good enough :smile:.